pillow_jxl/JpegXLImagePlugin.py

```diff
@@ -1,5 +1,9 @@
 from io import BytesIO
+
+import PIL
 from PIL import Image, ImageFile
+from pkg_resources import parse_version
```

Comment:
pkg_resources is deprecated: https://setuptools.pypa.io/en/latest/pkg_resources.html
and as setuptools isn't guaranteed to be installed, it may not even be there